import_streets/src/extract.rs

```diff
@@ -159,77 +146,7 @@ impl OsmExtract {
             return false;
         }
 
-        // Not sure what this means, found in Seoul.
-        if tags.is("lanes", "0") {
-            return false;
-        }
-
-        if opts.skip_local_roads && osm::RoadRank::from_highway(highway) == osm::RoadRank::Local {
```

Comment:
Also cleaning up an old hack that one user in A/B Street needed once. It could be useful to pretend a bunch of roads don't exist, but maybe doing it as some kind of later transformation on `StreetNetwork` would make more sense